cpp/ps0000/ps0000_solv.cpp

```diff
@@ -15,34 +16,38 @@ using namespace std;     // Missing
 */
 
 
-int main(void)          // incorrect
+// int Main(void)          // incorrect
 
-int main()             // correct
+int main(void)             // correct
 
 /* First error  : If you used void inside main function you must change int --> to be --> void too.
    Second error : It tells you that no value is returned for this program so the command return must be deleted .
-   Third error  : In this program you need to return a value so must delete void & put int .
+
+   1/1
 */
 
 {
-    cout >> "Hello , world! /n"   // incorrect
+    // cout >> "Hello , world! /n"   // incorrect
 
-    cout << "Hello , world! \n"; // correct
+    cout << "Hello, World! \n"; // correct
 
     /* First error  : The brackets used here are for the cin/input command, not the cout/output command.
        Second error : You are writing the wrong slant direction, it should be from this / to this \ .
        Third error  : You forgot the semicolon that is used to specify that this is the end of the requested command
                       and so that it moves to the next command.
+       3/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line